packages/db/src/types.ts

```diff
@@ -83,3 +83,25 @@ export interface PostgresProviderRecord extends PostgresProvider {
   updated_at: number,
   cumulusId: number,
 }
+
+export interface PostgresRule {
+  name: string,
+  workflow: string,
+  collection_cumulus_id: number,
+  provider_cumulus_id: number,
+  execution_name_prefix?: string,
+  type: string,
+  enabled: boolean,
+  value: string,
+  arn: string,
+  log_event_arn: string,
```

Comment:
I think these should all be optional based on the RDS schema

Reply:
Yes, you are correct